mlrun/api/db/sqldb/models.py

```diff
@@ -156,11 +159,30 @@ class Run(Base, HasStruct):
         start_time = Column(TIMESTAMP)
         labels = relationship(Label)
 
-    class Schedule(Base, HasStruct):
-        __tablename__ = "schedules"
+    class Schedule(Base):
+        __tablename__ = "schedules_v2"
```

Comment:
@Hedingber need to add state (active/standby/error/..)

Reply:
same